docs/web-apps/automated-testing/selenium/selenium4.md

```diff
@@ -30,15 +33,15 @@ Below are features new to Selenium 4 that may impact your tests.
 </tr>
 <tr>
 <td>Force W3C Usage</td>
-<td>Legacy JSON wire protocol support deprecated → uses the W3C WebDriver standard by default under the hood Force W3C usage (except Java). You can no longer use outdated DesiredCapabilities. No longer backwards compatible.</td>
+<td>Selenium 4 deprecates support for legacy JSON wire protocol and its static <code>DesiredCapabilities</code> browser methods, replacing it with <a href="/dev/w3c-webdriver-capabilities">W3C WebDriver-compliant protocol</a> and the use of <code>browserOptions</code>. You must completely switch from using JWP to W3C; Selenium will no longer be backwards-compatible. The W3C standard will be supported by default under the hood. </td>
```

Comment:
```suggestion
<td>Selenium 4 deprecates support for legacy JSON wire protocol and its static <code>DesiredCapabilities</code> browser methods, replacing it with <a href="/dev/w3c-webdriver-capabilities">W3C WebDriver-compliant protocol</a> and the use of <code>browserOptions</code>. You must completely switch from using JSON wire protocol to W3C; Selenium will no longer be backwards-compatible. The W3C standard will be supported by default under the hood. </td>
```

Reply:
static browser methods are completely gone in Se4